platform/android/java_godot_io_wrapper.cpp

```diff
@@ -118,6 +119,18 @@ int GodotIOJavaWrapper::get_screen_dpi() {
 	}
 }
 
+void GodotIOJavaWrapper::get_window_safe_area(int (&p_rect_xywh)[4]) {
+	if (_get_window_safe_area) {
+		JNIEnv *env = ThreadAndroid::get_env();
+		jintArray returnArray = (jintArray)env->CallObjectMethod(godot_io_instance, _get_window_safe_area);
+		jint *arrayBody = env->GetIntArrayElements(returnArray, JNI_FALSE);
```

Comment:
Could you add `ERR_FAIL_COND(env->GetArrayLength(returnArray) != 4)` here to be perfectly safe?

Reply:
Absolutely, that's a good idea.